oct/ansible/oct/roles/dependencies/tasks/register_repositories.yml

```diff
@@ -33,6 +33,26 @@
     - 3.5
   when: ansible_distribution == 'RedHat'
 
+- name: register RHEL 7 Atomic OpenShift Online Integration repositories
+  yum_repository:
+    name: 'rhel-7-server-online-int-{{ item }}-rpms'
+    state: present
+    description: 'A repository of dependencies for Atomic OpenShift {{ item }} Online'
+    baseurl: >
+      https://mirror.openshift.com/enterprise/online-int/latest/RH7-RHAOS-{{ item }}/x86_64/os/
+    gpgcheck: no
+    gpgkey: >
+      file:///etc/pki/rpm-gpg/RPM-GPG-KEY-redhat-release,
+      file:///etc/pki/rpm-gpg/RPM-GPG-KEY-redhat-beta,
+      https://mirror.ops.rhcloud.com/libra/keys/RPM-GPG-KEY-redhat-openshifthosted
```

Comment:
@bparees Since gpgcheck=0, I don't think key will be used. When we do get automated signing done, that key is correct.